packages/observability-node/src/cache.ts

```diff
@@ -0,0 +1,30 @@
+// TODO [PRO-629] Move this into the "shared" package.
```

Comment:
with the idea of shared pkg and context of PRO-629 (that isn't done) we should inline all content of this package into `/test-utils` and remove the explicit dependency on this package from `/test-utils`

Reply:
This way we'll be able to land this without finalizing the packaging strategy (that is the goal behind "meanwhile" section in PRO-629)